chromadb/server/fastapi/__init__.py

```diff
@@ -81,6 +82,15 @@ def use_route_names_as_operation_ids(app: _FastAPI) -> None:
             route.operation_id = route.name
 
 
+async def add_trace_id_to_response_middleware(
+    request: Request, call_next: Callable[[Request], Any]
+) -> Response:
+    trace_id = trace.get_current_span().get_span_context().trace_id
+    response = await call_next(request)
+    response.headers["Chroma-Trace-Id"] = format(trace_id, "x")
```

Comment:
ooc why hex

Reply:
the trace ID is internally represented as an int, Jaeger/Honeycomb both use the hex representation

Reply:
got it